config.py

```diff
@@ -24,6 +24,9 @@ class Config(object):
     SEFT_BUCKET_FILE_PREFIX = os.getenv("SEFT_BUCKET_FILE_PREFIX")
     SEFT_PUBSUB_PROJECT = os.getenv("SEFT_PUBSUB_PROJECT", "ras-rm-sandbox")
     SEFT_PUBSUB_TOPIC = os.getenv("SEFT_PUBSUB_TOPIC", "ras-rm-notify-test")
+    SEFT_CI_BUCKET_NAME = os.getenv("SEFT_CI_BUCKET_NAME")
+    GOOGLE_CLOUD_PROJECT = os.getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
This field might be okay to keep.  Can you change the bit that uses `SEFT_PUBSUB_PROJECT` to use this instead and remove the old value